diesel/src/query_builder/functions.rs

```diff
@@ -276,6 +276,50 @@ pub fn insert_into<T>(target: T) -> IncompleteInsertStatement<T, Insert> {
     IncompleteInsertStatement::new(target, Insert)
 }
 
+/// Creates an `INSERT [OR] IGNORE` statement.
+///
+/// If a constraint violation fails, the database will ignore the offending
+/// row and continue processing any subsequent rows. This function is only
+/// available with MySQL and SQLite.
+///
+/// With PostgreSQL, similar functionality is provided by [`on_conflict_do_nothing`].
+///
+/// [`on_conflict_do_nothing`]: ../diesel/query_builder/insert_statement/struct.InsertStatement.html#method.on_conflict_do_nothing
+///
+/// # Example
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("../doctest_setup.rs");
+/// #
+/// # #[cfg(not(feature = "postgres"))]
+/// # fn main() {
+/// #     use schema::users::dsl::*;
+/// #     use diesel::insert_or_ignore_into;
+/// #
+/// #     let conn = establish_connection();
+/// #     conn.execute("DELETE FROM users").unwrap();
```

Comment:
Even though this line is hidden, can we avoid using `conn.execute`? This is trivial in the public API: `diesel::delete(users).execute(&conn);`